packages/components/psammead-grid/README.md

```diff
@@ -0,0 +1,176 @@
+    </Grid>
+  </Grid>
+);
+```
+
+####<a name="multiple-groups">Setting values for multiple groups</a>
+
+```jsx
+import Grid from '@bbc/psammead-grid';
+
+const MyComponent = () => (
+  <Grid wrapper columns={{ group3: 6, group4: 8 }}>
+    <Grid columns={{ group3: 6, group4: 6 }}>
+      <p>Paragraph - for group 3 spans 6/6 columns, for group 4 spans 6/8 columns</p>
+    </Grid>
+    <Grid columns={{ group3: 6, group4: 2 }}>
+      <p>Paragraph - for group 3 spans 6/6 columns, for group 4 spans 2/8 columns</p>
+    </Grid>
+  </Grid>
+);
+```
+
+[Screenshot of this example without GEL Gutters and GEL Margins](./documentation/Screenshot\ example\ without\ gelMargins\ and\ gelGutters.png)
+
+####<a name="gutters-margins">Setting standard GEL gutters and GEL Margins</a>
+
+Using `enableGelGutters` and `enableGelMargins` on the wrapper Grid element.
+
+```jsx
+import Grid from '@bbc/psammead-grid';
+
+const MyComponent = () => (
+	<Grid
+		wrapper
+		enableGelGutters
+    enableGelMargins
+		columns={{ group3: 6, group4: 8 }}
+	>
+    <Grid columns={{ group3: 6, group4: 6 }}>
+      <p>Paragraph - for group 3 spans 6/6 columns, for group 4 spans 6/8 columns</p>
+    </Grid>
+    <Grid columns={{ group3: 6, group4: 2 }}>
+      <p>Paragraph - for group 3 spans 6/6 columns, for group 4 spans 2/8 columns</p>
+    </Grid>
+  </Grid>
+);
+```
+
+####<a name="nested-grid">Nested grid example</a>
+
+Note that here, for each definition of a parent Grid, we need to use the `wrapper` prop. This sets the total number of columns via the `columns` prop. Then to define how many columns the child should span, you can use the `columns` prop on the child `Grid`.
+
+```jsx
+import Grid from '@bbc/psammead-grid';
+
+const MyComponent = () => (
+  <Grid
+    wrapper
+    columns={{ group2: 6, group3: 6 }}
+  >
+		<Grid
+		  wrapper
+      columns={{ group2: 6, group3: 6 }}
+    >
+      <Grid columns={{ group2: 6, group3: 3 }}>
+        <ExampleImage />
+      </Grid>
+      <Grid columns={{ group2: 6, group3: 3 }}>
+        <ExampleParagraph />
+      </Grid>
+    </Grid>
+    <Grid columns={{ group2: 2, group3: 2 }}>
+      <ExampleImage />
+    </Grid>
+    <Grid columns={{ group2: 4, group3: 4 }}>
+      <ExampleParagraph />
+    </Grid>
+  </Grid>
+);
+```
+
+[Screenshot of this example with GEL Gutters and GEL Margins enabled](./documentation/Screenshot\ example\ with\ gelMargins\ and\ gelGutters.png)
+
+** Setting GEL Max Widths**
+At the page level, you may want to have the GEL Grid's fixed max widths. 
+You can enable it with the prop `enableGelMaxWidths`. 
+
+For some pages, we will want to constrain the page to 1008px. You can use `enableGroupFourMaxWidth` for this. 
+
+### When to use this component
+
+This `Grid` component can be used for page-level layouts as well as for layouts of individual components. You can nest Grids within one another for more complex layouts. Please see the Storybook Stories for examples.
+
+### Frequenty Asked Questions
+
```

Comment:
I don't know if this is the best place for this question, but I think it'd be good to clarify what happens when you have grid items with a total number of columns that's greater than the wrapper they're in, eg:

```jsx
<Grid wrapper columns={{ group2: 4 }}>
  <Grid columns={{ group2: 3 }}>
    <Child />
  </Grid>
  <Grid columns={{ group2: 2 }}>
    <Child />
  </Grid>
</Grid>
```
I'm guessing the second non-wrapper Grid will move onto a second row and leave the 4th column of the first row empty. Is that correct?

Reply:
Yes that is correct. I've created this diagram to show what happens in that case.
The first item spans the three columns and the second item spans two on the next row down
<img width="625" alt="four column grid with items three and two" src="https://user-images.githubusercontent.com/3028997/65766642-8734cc00-e123-11e9-96f6-23ad16d627a0.png">